openfe/protocols/openmm_afe/equil_solvation_afe_method.py

```diff
@@ -397,13 +396,15 @@ def _default_settings(cls):
           a set of default settings
         """
         return AbsoluteSolvationSettings(
-            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(),
+            protocol_repeats=3,
+            solvent_forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(),
+            vacuum_forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(
```

Comment:
We'll need new validators to ensure that the FF settings haven't diverged, please raise an issue.